components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/client/ApplicationManagementServiceClient.java

```diff
@@ -274,6 +276,140 @@ public String exportApplication(String appid, boolean exportSecrets) throws Axis
         return null;
     }
 
+    /**
+     * Create an application template.
+     *
+     * @param spTemplate service provider template info
+     * @throws AxisFault
+     */
+    public void createApplicationTemplate(SpTemplate spTemplate) throws AxisFault {
+
+        try {
+            if (debugEnabled) {
+                log.debug(String.format("Registering Service Provider template: %s", spTemplate.getName()));
```

Comment:
We can just use string concat, instead of String.format here. It is much efficient.